rootfs-builder/fedora/rootfs_lib.sh

```diff
@@ -43,7 +43,17 @@ build_rootfs()
 
 	#In case rootfs is created usig repositories allow user to modify
 	# the default URL
-	local REPO_URL=${REPO_URL:-http://mirror.math.princeton.edu/pub/fedora/linux/releases/$OS_VERSION/Everything/x86_64/os/}
+        case ${ARCH} in
+                x86_64|armhfp)
+                    local REPO_URL=${REPO_URL:-http://mirror.math.princeton.edu/pub/fedora/linux/releases/$OS_VERSION/Everything/$ARCH/os/}
```

Comment:
Looking at a F27 system, their convention is to have their mirrors determine the correct URL:

```
https://mirrors.fedoraproject.org/metalink?repo=updates-released-f$releasever&arch=$basearch
```

I think this script could use the same technique to avoid having to case the architecture?

Reply:
Yes, makes sense. I will update the PR.

Reply:
Thanks @harche.